feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/fee/utils/HydraDxQuoteSharedComputation.kt

```diff
@@ -35,31 +36,42 @@ class HydraDxQuoteSharedComputation(
         }
     }
 
-    suspend fun quote(
+    suspend fun paths(
         chain: Chain,
+        args: AssetExchangeQuoteArgs,
         accountId: AccountId,
-        fromAsset: Chain.Asset,
-        toAsset: Chain.Asset,
         scope: CoroutineScope
-    ): AssetExchangeQuote {
-        val key = "HydraDxQuote:${accountId.toHexString()}"
+    ): List<Path<HydraDxSwapEdge>> {
+        val key = "HydraDxPaths:${accountId.toHexString()}"
 
         return computationalCache.useCache(key, scope) {
-            val args = AssetExchangeQuoteArgs(
-                chainAssetIn = fromAsset,
-                chainAssetOut = toAsset,
-                amount = BigInteger.ZERO,
-                swapDirection = SwapDirection.SPECIFIED_IN
-            )
-
             val assetConversion = getAssetConversion(chain, accountId, scope)
             val swapDirections = directions(chain, accountId, scope)
-            val paths = assetConversion.getPaths(swapDirections, args)
-            assetConversion.quote(paths, args)
+            assetConversion.getPaths(swapDirections, args)
         }
     }
 
-    private suspend fun getAssetConversion(
+    suspend fun quote(
```

Comment:
Why do we have two `quote` implementations and dublicated code of this implementation in ` HydraDxQuoteSharedComputation`

Reply:
Right. Thank you. I forgot to remove this one